### ISSUE TITLE
import/export functions

### ISSUE THREAD

**Issue by dom-mel** [dokufreaks/dokuwiki-translation#7]

**Comment by Klap-in** [comment]
@dom-mel What do you mean by this issue?

**Comment by splitbrain** [comment]
I think the idea here was to let people export the current translation in some common format (not sure what format), let them edit the translation in some editor and then reupload it.

**Comment by Klap-in** [comment]
In the view template I discover some forms
https://github.com/dokufreaks/dokuwiki-translation/blob/master/src/org/dokuwiki/translatorBundle/Resources/views/Default/show.html.twig#L45

which can be enabled/disabled via the config https://github.com/dokufreaks/dokuwiki-translation/blob/master/app/config/parameters.yml.dist#L57

But sofar indeed no processing in the backend founded.

**Comment by dom-mel** [comment]
Yes this is the idea. There are some common formats for languages. 

The idea is to let translators use their normal translation tools
